spice/client-tests/zoom.py

```diff
@@ -38,6 +38,8 @@
                    const="mouse", dest="method")
 group.add_argument("-w", "--windowmanager", help="Use window manager.",
                    action="store_const", const="wm_key", dest="method")
+parser.add_argument("-r", "--repeatreturn", help="Repeat zoom and return to normal.", default=0, 
```

Comment:
In the best way there are should be two options:
--repeat
--reset

They are two different actions. The module will be more reusable.

If you have time, please enhance in this way.

Thank you.